include/toplevel.mk

```diff
@@ -215,6 +215,9 @@ check: .config FORCE
 val.%: FORCE
 	@+$(NO_TRACE_MAKE) -r -s $@ QUIET= V=s
 
+var.%: FORCE
+	@+$(NO_TRACE_MAKE) -r -s $@ QUIET= V=s
+
```

Comment:
Couldn't these targets simply be combined? Makes it obvious their handling is the same... 
```
val.% var.%: FORCE
	@+$(NO_TRACE_MAKE) -r -s $@ QUIET= V=s
```

Reply:
yeah good idea, I reduced it to a 1 line diff